text.go

```diff
@@ -13,10 +15,160 @@ const (
 	TextAlignTrailing
 )
 
+// TextWrap represents how text longer than the widget's width will be wrapped.
+type TextWrap int
+
+const (
+	// TextWrapOff extends the widget's width to fit the text, no wrapping is applied.
+	TextWrapOff TextWrap = iota
+	// TextTruncate trims the text to the widget's width, no wrapping is applied.
+	TextTruncate
+	// TextWrapBreak trims the line of characters to the widget's width adding the excess as new line.
+	TextWrapBreak
+	// TextWrapWord trims the line of words to the widget's width adding the excess as new line.
+	TextWrapWord
+)
+
 // TextStyle represents the styles that can be applied to a text canvas object
 // or text based widget.
 type TextStyle struct {
 	Bold      bool // Should text be bold
 	Italic    bool // Should text be italic
 	Monospace bool // Use the system monospace font instead of regular
 }
+
+// MeasureText uses the current driver to calculate the size of text when rendered.
+func MeasureText(text string, size int, style TextStyle) Size {
+	return CurrentApp().Driver().RenderedTextSize(text, size, style)
+}
+
+// SplitLines accepts a slice of runes and returns a slice containing the
+// start and end indicies of each line delimited by the newline character.
+func SplitLines(text []rune) [][2]int {
```

Comment:
This was mentioned before but I don't think we discussed it...
New APIs should only be introduced as required and I don't think these are needed.

Reply:
Ack, I'll unexport it - its signature will change in future anyway with the rest of the text improvements.